packages/frontend/core/src/modules/cloud/services/servers.ts

```
@@ -47,6 +47,17 @@ export class ServersService extends Service {
     [] as any
   );
 
+  serversWithAccount$ = this.servers$
```

ESLint

[error] Finnish notation should not be used here.

(rxjs/finnish)